mantidproject/mantid
# ISIS-Powder POLARIS requirements for next cycle

Following discussions with POLARIS, a list of adjustments to be made to their `isis_powder` scripts was drawn up [here](https://github.com/mantidproject/mantid/issues/20603). This PR covers these changes.

**To test:**

Validation with POLARIS has taken place, so code review and check system & unit tests pass should be fine. If you want to try and break things, come and see me and I'll give you a hand setting up the `isis_powder` directories, as they take a little bit of fiddling

Fixes #20603 . 

**Release Notes** 

[Here](https://github.com/mantidproject/mantid/compare/20603_polaris_requirements_3_12#diff-6d316275f01d64b8113c082a773943ae)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
